backend/src/routes/v1/kcClientForm.js

```diff
@@ -0,0 +1,64 @@
+const config = require('config');
+const cryptico = require('cryptico-js');
+const log = require('npmlog');
+
+const RealmAdminService = require('../../components/realmAdminSvc');
+const KeyCloakServiceClientManager = require('../../components/keyCloakServiceClientMgr');
+
+const {
+  body,
+  validationResult
+} = require('express-validator');
+
+// submits a keycloak service client
+// will need to add public key to this for encryption from frontend...
+const kcClientForm = require('express').Router();
+kcClientForm.post('/', [
```

Comment:
Missing whitespace line above

Reply:
are these enforceable in es-lint?

Reply:
addressed in next commit

Reply:
I don't think there's anything in es-lint regarding the above type of spacing unfortunately - at least I'm not aware of it anyways.